root/usr/share/nethesis/NethServer/Module/IpsecTunnels.php

```diff
@@ -55,4 +58,25 @@ public function initialize()
         parent::initialize();
     }
 
+    private function readStatus()
+    {
+        if (!$this->status) {
```

Comment:
Hint: a PHP static variable would limit the scope of "status" to this function only